lib/src/platform_client.dart

```diff
@@ -62,6 +64,22 @@ class PlatformClient {
     await _httpPost('/api/smartapp/verify', body);
   }
 
+  /// Confirms a verification code that was sent to a phone number by [sendPhoneVerificationCode].
+  ///
+  /// [phoneNumber] is the phone number in E.164 format and [verificationCode] is the verification code. If successful,
+  /// the returned [Credentials] can be used to [createAccount] (if [Credentials.isNewUser] is `true`) or to
+  /// [loginWithCredentials].
+  Future<Credentials> confirmPhoneVerificationCode(String phoneNumber, String verificationCode) async {
+    String body = jsonEncode({
+      'authCode': verificationCode,
+      'phoneNumber': phoneNumber,
+    });
+
+    Map<String, dynamic> response = await _httpPost('/api/smartapp/verify/confirm', body);
+
+    return Credentials('PHONE_VERIFICATION', phoneNumber, response['verificationCode'], response['newUser']);
```

Comment:
For these const values like `PHONE_VERIFICATION`, I usually prefer to see them defined as a static const variable to avoid typos which your eyes have a hard time to see and for which the compiler could help...

In this case though, it would not be used almost at all... keep it as is.